src/Lumper.Lib/BSP/Struct/Entity.cs

```diff
@@ -49,7 +49,7 @@ public object Clone()
 
     public abstract class EntityProperty(string key) : ICloneable
     {
-        public string Key { get; } = key;
+        public string Key { get; set; } = key;
```

Comment:
this should is probably also a fixup for the initial commit (I realize now that this is the stuff from the fixup branch but the commit message doesn't say fixup, so I'll point it out)

Reply:
When it's a clear fix to an existing thing I think keeping these split should be fine unless they make rebasing harder